documentation/source/conf.py

```diff
@@ -12,21 +12,25 @@
 #
 import os
 import sys
-sys.path.insert(0, os.path.abspath('../../src'))
+
+# Imports for auto documentation
+import super_gradients
+import super_gradients.common as common
+import super_gradients.training as training
+
+__all__ = ["super_gradients", "common", "training"]
+
+sys.path.insert(0, os.path.abspath("../../src"))
 
 # -- Project information -----------------------------------------------------
 
-project = 'SuperGradients'
-copyright = '2021, SuperGradients team'
-author = 'SuperGradients team'
+project = "SuperGradients"
+copyright = "2021, SuperGradients team"
+author = "SuperGradients team"
 
 # The full version, including alpha/beta/rc tags
-release = '1.0'
+release = "1.0"
```

Comment:
Change to 3.0.2 ?

Reply:
fixed in [e3a13fc](https://github.com/Deci-AI/super-gradients/pull/462/commits/e3a13fc0bc2ff7ba0dbc34c64291693e99109559)